frontend/src/store/index.js

```diff
@@ -46,7 +62,257 @@ export function* watchSigninRequest() {
 }
 
 
+
+
+
+
+
+
+/*
+**********************
+* *******MEETING******
+* ********************
+ */
+
+
+
+// GET 'meetings/id'
+export function* getMeetingRequest(token, index) {
+
+	const { status, data } = yield call(api.get, backendUrl+'meetings/'+index+ '/', token)
```

Comment:
it's better to call with  `` strings. will be optimized ( i.e. `{${backendUrl}/meetings/${index}/` )